HW1_Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Стих
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ReadLine();
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Сергей Есенин Мои мечты");
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine("Мои мечты стремятся вдаль,");
+            Console.WriteLine ("Чужую разделить печаль");
+            Console.WriteLine("Где слышны вопли и рыданья");
+            Console.WriteLine( "И муки тяжкого страданья.");
+            Console.WriteLine( "Я там могу найти себе");
+            Console.WriteLine("Отраду в жизни, упоенье,");
+            Console.WriteLine ("И там, наперекор судьбе,");
+            Console.WriteLine("Искать я буду вдохновенья") ;
+            
+            
```

Comment:
А здесь уместно ожидание нажатия на клавишу `Console.ReaKey()` (а не ввод строки `Console.ReadLine()` ), чтобы после окончания работы программы в отладочном режиме консоль сразу не закрылась.
```suggestion
            Console.ReaKey();
```